### DISCUSSION TITLE
Does it work on Compose Multiplatform applications?

### DISCUSSION THREAD

**Discussion by MessiasLima** [hi-manshu/Kalendar#87] [General]
I'm trying to use it in a compose multiplatform application but the `Kalendar` composable can't be imported.

Is this library supposed to work on Kotlin Multiplatform applications?

**Comment by hi-manshu** [comment]
Hey, thank you for coming to herE
Actually, it does not support Kotlin multiplatform yet as it was made for native applications


But will try to add support sooner.

**Reply by MessiasLima** [reply]
Thank you for your answer

**Comment by hi-manshu** [comment]
Hey it went too late, but now it's there! You can check it
